driver/ppm_events_public.h

```diff
@@ -418,6 +418,12 @@ along with sysdig.  If not, see <http://www.gnu.org/licenses/>.
 #define PPM_IPC_NOWAIT		(1 << 0)
 #define PPM_SEM_UNDO		(1 << 1)
 
+/*
+ * Semget flags
+ */
+#define PPM_IPC_CREAT  (1 << 13)
+#define PPM_IPC_EXCL   (1 << 14)
+
```

Comment:
Ok I got your point. I read also on the include files that those constants have the same prefix but are used by different syscalls. I think it's ok as it is for now!